compiler/builders/uvg_builder.jou

```diff
@@ -10,29 +9,37 @@ import "./ast_to_builder.jou"
 import "./either_builder.jou"
 
 
-# Within this class, UVG variable ID -1 denotes a value that is not a pointer,
-# or is some pointer that we don't keep track of. Any other ID represents a
-# pointer to the corresponding UVG variable.
+# Within this class, we use the following special UVG variable IDs.
+# TODO: ideally these would be constants, not functions.
+
+# A value that is not a pointer, or is some pointer that we don't keep track of.
+def ANONYMOUS_VALUE_ID() -> int:
+    return -1
+
+# Boolean value known to be true at compile time.
+def TRUE_ID() -> int:
+    return -2
+
+# Boolean value known to be false at compile time.
+def FALSE_ID() -> int:
+    return -3
+
+
 class UBuilder:
     uvg: Uvg
     current_block: UvgBlock*
-    returns_a_value: bool
     location: Location
+    warn_unreachable: bool
```

Comment:
Do we really need this in this PR? This PR should only check undefined values, not unreachable code.

Reply:
No